src/Package/Impl/Publishing/Handlers/RmdPublishHandler.cs

```diff
@@ -28,7 +29,7 @@ internal sealed class RmdPublishHandler : IMarkdownFlavorPublishHandler {
             string outputFile = Path.GetFileName(outputFilePath);
             string outputFolder = Path.GetDirectoryName(outputFilePath).Replace('\\', '/');
             // Run rmarkdown::render
-            return $"\"rmarkdown::render(\'{inputFile}\', output_format=\'{format}\', output_file='{outputFile}',  output_dir='{outputFolder}')\"";
```

Comment:
This is generated command that we invoke so we specify encoding that we save the file with. However, it is a good question if someone tries to do the same manually. In this case they won't be going via this path and rather would need to a) save file in VS with desired encoding and b) specify that encoding in the command line. Although I guess no one would ever publish RMD manually.